src/examples/03_custom_mobjects/screen_grid.py

```diff
@@ -0,0 +1,99 @@
+            self.add(Line(
+                [x - self.width / 2., -self.height / 2., 0],
+                [x - self.width / 2., self.height / 2., 0],
+            ))
+        for y in np.arange(0, self.height + y_step, y_step):
+            self.add(Line(
+                [-self.width / 2., y - self.height / 2., 0],
+                [self.width / 2., y - self.height / 2., 0]
+            ))
+
+
+class ScreenGrid(VGroup):
+    CONFIG = {
+        "rows": 8,
+        "columns": 14,
+        "height": FRAME_Y_RADIUS * 2,
+        "width": 14,
+        "grid_stroke": 0.5,
+        "grid_color": WHITE,
+        "axis_color": RED,
+        "axis_stroke": 2,
+        "show_points": False,
+        "point_radius": 0,
+        "labels_scale": 0.5,
+        "labels_buff": 0,
+        "number_decimals": 2
+    }
+
+    def __init__(self, **kwargs):
+        VGroup.__init__(self, **kwargs)
+        rows = self.rows
+        columns = self.columns
+        grilla = Grid(width=self.width, height=self.height, rows=rows, columns=columns)
+        grilla.set_stroke(self.grid_color, self.grid_stroke)
+
+        vector_ii = ORIGIN + np.array((- self.width / 2, - self.height / 2, 0))
+        vector_si = ORIGIN + np.array((- self.width / 2, self.height / 2, 0))
+        vector_sd = ORIGIN + np.array((self.width / 2, self.height / 2, 0))
+
+        ejes_x = Line(LEFT * self.width / 2, RIGHT * self.width / 2)
+        ejes_y = Line(DOWN * self.height / 2, UP * self.height / 2)
+
+        ejes = VGroup(ejes_x, ejes_y).set_stroke(self.axis_color, self.axis_stroke)
+
+        divisiones_x = self.width / columns
+        divisiones_y = self.height / rows
+
+        direcciones_buff_x = [UP, DOWN]
+        direcciones_buff_y = [RIGHT, LEFT]
+        dd_buff = [direcciones_buff_x, direcciones_buff_y]
+        vectores_inicio_x = [vector_ii, vector_si]
+        vectores_inicio_y = [vector_si, vector_sd]
+        vectores_inicio = [vectores_inicio_x, vectores_inicio_y]
+        divisiones = [divisiones_x, divisiones_y]
+        orientaciones = [RIGHT, DOWN]
+        puntos = VGroup()
+        leyendas = VGroup()
+        set_changes = zip([columns, rows], divisiones, orientaciones, [0, 1], vectores_inicio, dd_buff)
+        for tipo, division, orientacion, coordenada, vi_c, d_buff in set_changes:
+            for i in range(1, tipo):
+                for v_i, direcciones_buff in zip(vi_c, d_buff):
+                    ubicacion = v_i + orientacion * division * i
+                    punto = Dot(ubicacion, radius=self.point_radius)
+                    coord = round(punto.get_center()[coordenada], self.number_decimals)
+                    leyenda = TextMobject("%s" % coord).scale(self.labels_scale)
+                    leyenda.next_to(punto, direcciones_buff, buff=self.labels_buff)
+                    puntos.add(punto)
+                    leyendas.add(leyenda)
+
+        self.add(grilla, ejes, leyendas)
+        if self.show_points:
+            self.add(puntos)
```

Comment:
Setting `show_points` to `True` doesn't change anything. I saw that it is because the radius of the points is 0. Maybe set it to a value > 0 so it is not so confusing when changing the option and nothing happens.